i18n/en.json

```diff
@@ -4198,6 +4206,50 @@
     "id": "ent.cluster.stopping.info",
     "translation": "Cluster internode communication is stopping on %v with hostname=%v id=%v"
   },
+  {
+    "id": "ent.compliance.csv.attachment.export.appError",
+    "translation": "Unable to add attachment to the CSV export."
+  },
+  {
+    "id": "ent.compliance.csv.attachment.copy.appError",
+    "translation": "Unable to copy the attachment into the zip file."
+  },
+  {
+    "id": "ent.compliance.csv.file.creation.appError",
+    "translation": "Unable to create CSV export temporary file."
+  },
+  {
+    "id": "ent.compliance.csv.header.export.appError",
+    "translation": "Unable to add the header to the CSV export."
+  },
+  {
+    "id": "ent.compliance.csv.metadata.export.appError",
+    "translation": "Unable to add the metadata file to the zip file."
+  },
+  {
+    "id": "ent.compliance.csv.metadata.json.marshalling.appError",
+    "translation": "Unable to marshall the metadata of the CSV export."
+  },
+  {
+    "id": "ent.compliance.csv.post.export.appError",
+    "translation": "Unable to add post to the CSV export."
+  },
+  {
+    "id": "ent.compliance.csv.zip.creation.appError",
+    "translation": "Unable to create CSV export temporary zip."
```

Comment:
`Unable to create temporary CSV export zip file.`

Reply:
Fixed